azext_iot/tests/dps/test_iot_dps_int.py

```diff
@@ -101,6 +102,13 @@ def __init__(self, test_method):
         self.cmd_shell = Setting()
         setattr(self.cmd_shell, "cli_ctx", self.cli_ctx)
 
+        if hub != os.environ.get("azext_iot_testhub"):
```

Comment:
IoT Hub tests uses `DynamoSettings` and the same could be used here to support uniform test plumbing across technology areas.